clang/test/Driver/cheri/bounds-mode.c

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: mips-registered-target
```

Comment:
Shouldn't need a registered target?

Reply:
Same issue as below with -mxcaptable

Reply:
Hmm this may be slightly annoying to fix properly, so just adding this requires seems fine to me.

Reply:
Well, this test isn't really a Driver test, as it's using -emit-llvm

Reply:
That's fair, ideally `-emit-llvm` should be removed. Maybe the flag validation only happens in the backend but it's been a long time since I looked at that code. I wonder if -E also verifies it?

Reply:
I could be wrong here. But the tests here invoke `clang` directly, which invokes the driver too. If we're just testing frontend code I think these tests should be modified to only invoke `clang -cc1`